doc/introduction/pennylane.rst

```diff
@@ -39,14 +39,25 @@ through the computation - an essential process in optimization and machine learn
 Learn more
 ----------
 
-In the following sections you can learn more about quantum circuits and interfaces in PennyLane:
+In the following sections you can learn more about the key features of PennyLane:
 
-1. :ref:`Quantum circuits <intro_vcircuits>` shows you how PennyLane unifies and
-simplifies the process of programming quantum circuits with trainable parameters.
+1. :ref:`Quantum circuits <intro_vcircuits>` shows how PennyLane unifies and simplifies 
+the process of programming quantum circuits with trainable parameters.
 
-2. :ref:`Interfaces <intro_interfaces>` introduces you to how PennyLane is used
-with the different classical interfaces to optimize quantum circuits or hybrid computations.
+2. :ref:`Interfaces <intro_interfaces>` introduces how PennyLane is used with the different 
+classical interfaces to optimize quantum circuits or hybrid computations.
```

Comment:
Classical interfaces might not mean much to people exploring PennyLane for the first time
```suggestion
machine learning libraries to optimize quantum circuits or hybrid computations.
```